hangman.js

```diff
@@ -1 +1,40 @@
-console.log("welcome to hangman");
\ No newline at end of file
+console.log("welcome to hangman");
+
+var game = {
+  word: "",
+  letters: [],
+  playerScore: 0,
+  executionerScore: 0,
+
+
+  getWord: function() {
+    this.word = prompt("Please enter the word that the player will guess.");
+    this.letters = this.word.split("");
+    this.guessLetter();
+  },
+
+  guessLetter: function() {
+    var guess = prompt("Please guess a letter");
+    if (this.letters.includes(guess)) {
+      for (i = 0; i < this.letters.length; i++) {
+        if (guess === this.letters[i]) {
+          this.playerScore++;
+          }
+        }
+      }
+    else {
+      this.executionerScore++;
+    }
+    this.checkScore();
+  },
+
+  checkScore: function() {
+    alert("Your score is " + this.playerScore + ", and the executioner's score is " + this.executionerScore +".");
+    if (this.playerScore === this.letters.length) {
+      return alert("Game over. You win!");
+    } else if (this.executionerScore === 6) {
+      return alert("Game over, maaaan! You shall hang by the neck until you are dead, dead, dead.");
+    } else this.guessLetter();
```

Comment:
Interesting, I like this back and forth between `guessLetter` and `checkScore`.

Think about other ways you could implement this same functionality (e.g., wrapping a function/the game in a while loop, recursions).